hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/SignatureProcessor.java

```diff
@@ -32,7 +33,7 @@
   String X_AMAZ_DATE = "X-Amz-Date";
   String CONTENT_MD5 = "content-md5";
   String AUTHORIZATION_HEADER = "Authorization";
-  Charset UTF_8 = Charset.forName("utf-8");
+  Charset UTF_8 = StandardCharsets.UTF_8;
```

Comment:
Nit: usages of this constant can be replaced with the one from `StandardCharsets`.